interface/src/components/dialogs/newcontainer_dialog/NewContainerDialog.component.tsx

```diff
@@ -90,14 +120,22 @@ function NewContainerDialog({ open, handleClose, dialogTitle, servers }: any) {
         open={open}
         onClose={handleClose}
         aria-labelledby="form-dialog-title"
-        onKeyPress={(event) => (event.key === "Enter" ? handleConfirmation() : null)}
+        onKeyPress={(event) =>
+          event.key === "Enter" ? handleConfirmation() : null
+        }
       >
         <DialogTitle id="form-dialog-title">{dialogTitle}</DialogTitle>
         <DialogContent>
           <form noValidate autoComplete="off">
             <Grid container spacing={2}>
-              <Grid item xs={12} style={{ padding: "2px", paddingLeft: "12px" }}>
-                <Typography variant="subtitle2">Server to Start Container On</Typography>
+              <Grid
+                item
+                xs={12}
+                style={{ padding: "2px", paddingLeft: "12px" }}
+              >
```

Comment:
I'm not sure that I agree that this has to be exploded into more lines.

Reply:
Agreed, no need to.